tests/workflows/test_uber_lyft.py

```diff
@@ -0,0 +1,67 @@
+import uuid
+
+import coiled
+import dask.dataframe as dd
+import pytest
+from dask.distributed import Client
+
+
+@pytest.fixture(scope="module")
+def uber_lyft_cluster(
+    dask_env_variables,
+    cluster_kwargs,
+    github_cluster_tags,
+):
+    with coiled.Cluster(
+        f"uber-lyft-{uuid.uuid4().hex[:8]}",
+        environ=dask_env_variables,
+        tags=github_cluster_tags,
+        **cluster_kwargs["uber_lyft_cluster"],
+    ) as cluster:
+        yield cluster
+
+
+@pytest.fixture
+def uber_lyft_client(
+    uber_lyft_cluster,
+    cluster_kwargs,
+    upload_cluster_dump,
+    benchmark_all,
+):
+    n_workers = cluster_kwargs["uber_lyft_cluster"]["n_workers"]
+    with Client(uber_lyft_cluster) as client:
+        uber_lyft_cluster.scale(n_workers)
+        client.wait_for_workers(n_workers)
+        client.restart()
+        with upload_cluster_dump(client), benchmark_all(client):
+            yield client
+
+
+@pytest.fixture
+def ddf(uber_lyft_client, read_parquet_with_pyarrow):
+    """NYC taxi Uber/Lyft dataset"""
+    return dd.read_parquet(
+        "s3://coiled-datasets/uber-lyft-tlc/",
+        engine="pyarrow",
```

Comment:
Should be able to drop this since `"pyarrow"` is the default engine (when `pyarrow` is installed) 

```suggestion
```